modules/riseBidAdapter.js

```diff
@@ -147,11 +147,11 @@ function getFloor(bid, mediaType) {
     return 0;
   }
   let floorResult = bid.getFloor({
-    currency: CURRENCY,
+    currency: bid.params.currency || DEFAULT_CURRENCY,
```

Comment:
1. please select from 3 options
2. extract the currency selection to a function or a var

Reply:
Done